plugins/source/aws/resources/services/elasticache/gen.hcl

```diff
@@ -123,4 +123,575 @@ resource "aws" "elasticache" "clusters" {
+  }
+
+  multiplex "AwsAccountRegion" {
+    path   = "github.com/cloudquery/cloudquery/plugins/source/aws/client.ServiceAccountRegionMultiplexer"
+    params = ["elasticache"]
+  }
+
+  options {
+    primary_keys = [
+      "account_id",
+      "region",
+      "name"
+    ]
+  }
+
+  userDefinedColumn "account_id" {
+    description = "The AWS Account ID of the resource."
+    type        = "string"
+    resolver "resolveAWSAccount" {
+      path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.ResolveAWSAccount"
+    }
+  }
+
+  userDefinedColumn "region" {
+    type        = "string"
+    description = "The AWS Region of the resource."
+    resolver "resolveAWSRegion" {
+      path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.ResolveAWSRegion"
+    }
+  }
+
+  column "service_update_name" {
+    rename = "name"
+  }
+
+  column "service_update_description" {
+    rename = "description"
+  }
+  
+  column "service_update_end_date" {
+    rename = "end_date"
+  }
+
+  column "service_update_recommended_apply_by_date" {
+    rename = "recommended_apply_by_date"
+  }
+
+  column "service_update_release_date" {
+    rename = "relase_date"
+  }
+
+  column "service_update_severity" {
+    rename = "severity"
+  }
+
+  column "service_update_status" {
+    rename = "status"
+  }
+
+  column "service_update_type" {
+    rename = "type"
+  }
+
+}
+
+resource "aws" "elasticache" "snapshots" {
+  path = "github.com/aws/aws-sdk-go-v2/service/elasticache/types.Snapshot"
+
+  ignoreError "IgnoreCommonErrors" {
+    path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.IgnoreCommonErrors"
+  }
+
+  deleteFilter "AccountRegionFilter" {
+    path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.DeleteAccountRegionFilter"
+  }
+
+  multiplex "AwsAccountRegion" {
+    path   = "github.com/cloudquery/cloudquery/plugins/source/aws/client.ServiceAccountRegionMultiplexer"
+    params = ["elasticache"]
+  }
+
+  options {
+    primary_keys = [
+      "arn"
+    ]
+  }
+
+  userDefinedColumn "account_id" {
+    description = "The AWS Account ID of the resource."
+    type        = "string"
+    resolver "resolveAWSAccount" {
+      path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.ResolveAWSAccount"
+    }
+  }
+
+  userDefinedColumn "region" {
+    type        = "string"
+    description = "The AWS Region of the resource."
+    resolver "resolveAWSRegion" {
+      path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.ResolveAWSRegion"
+    }
+  }
+
+  column "auto_minor_version_upgrade" {
+    description = "Auto minor version upgrade"
+  }
+
+  column "data_tiering" {
+    description = "Data tiering"
+  }
+
+  column "node_group_configuration_node_group_id" {
+    rename = "node_group_id"
+  }
+
+  column "node_group_configuration_primary_availability_zone" {
+    rename = "node_group_primary_availability_zone"
+  }
+
+  column "node_group_configuration_primary_outpost_arn" {
```

Comment:
Should this be `output` instead of `outpost`?

Reply:
nope :)
https://aws.amazon.com/outposts/

Reply:
@shimonp21 Oh, in that case, the renamed column below needs to be fixed--it says `output` instead of `outpost`